.circleci/config.yml

```diff
@@ -505,6 +507,18 @@ jobs:
       - run: CC=gcc-8 CXX=g++-8 V=1 make -j16 all
       - post-steps
 
+  build-linux-cmake-with-folly-coroutines:
+    machine:
+      image: ubuntu-2004:202111-02
+    resource_class: 2xlarge
+    steps:
+      - pre-steps
+      - run: sudo apt-get update -y && sudo apt-get install gcc-10 g++-10 libgflags-dev
+      - setup-folly
```

Comment:
Under the paradigm of folly being an external library, it would be nice if one shared CircleCI step downloaded & compiled folly, though we would probably need to add CC and CXX to the `environment` for this build to make that work (which would reduce copy-paste anyway).